fbw-common/src/systems/shared/src/publishers/Extras/GsxSimVarPublisher.ts

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) 2021-2024 FlyByWire Simulations
+// SPDX-License-Identifier: GPL-3.0
+
+import { EventBus, PublishPacer, SimVarDefinition, SimVarPublisher, SimVarValueType } from '@microsoft/msfs-sdk';
+
+/* eslint-disable camelcase */
```

Comment:
Do you really need this? I think we removed that rule some time ago.

Reply:
Removed from GsxSimVarPublisher.ts
Housekeeping the other Files would just bloat this PR imho (more than it already is because I couldn't stop ^^)